Build error: "'aclocal-1.14' is missing on your system."

`cargo build` runs fine on my machine, but in `cargo build --target arm-linux-androideabi` the build script fails with stderr as below. What’s `aclocal-1.14`? Do I need to install something?

CC @larsbergstrom @mbrubeck 

```
make[1]: warning: -jN forced in submake: disabling jobserver mode.
/home/simon/projects/servo/deps/rust-harfbuzz/harfbuzz-sys/harfbuzz/missing: line 81: aclocal-1.14: command not found
WARNING: 'aclocal-1.14' is missing on your system.
         You should only need it if you modified 'acinclude.m4' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'aclocal' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
         It also requires GNU Autoconf, GNU m4 and Perl in order to run:
         <http://www.gnu.org/software/autoconf>
         <http://www.gnu.org/software/m4/>
         <http://www.perl.org/>
make[1]: *** [/home/simon/projects/servo/deps/rust-harfbuzz/harfbuzz-sys/harfbuzz/aclocal.m4] Error 127
make: *** [all] Error 2
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,6 @@ rust:
  - stable
  - nightly
 script:
- - touch harfbuzz-sys/harfbuzz/{configure.ac,aclocal.m4,configure,Makefile.am,Makefile.in}
  - cargo build --verbose --manifest-path=harfbuzz-sys/Cargo.toml
 notifications:
   webhooks: http://build.servo.org:54856/travis

--- a/harfbuzz-sys/Cargo.toml
+++ b/harfbuzz-sys/Cargo.toml
@@ -2,6 +2,16 @@
 name = "harfbuzz-sys"
 version = "0.1.2"
 
+# Ignore timestamps on automake-generated files. See comment in makefile.cargo.
+exclude = [
+    "harfbuzz/configure.ac",
+    "harfbuzz/aclocal.m4",
+    "harfbuzz/configure",
+    "harfbuzz/config.h.in",
+    "harfbuzz/Makefile.am",
+    "harfbuzz/Makefile.in",
+]
+
 authors = ["The Servo Project Developers"]
 license = "MIT"
 

--- a/harfbuzz-sys/makefile.cargo
+++ b/harfbuzz-sys/makefile.cargo
@@ -28,9 +28,21 @@ CONFIGURE_FLAGS = \
 	--without-glib
 
 all:
+	touch $(AUTOMAKE_FILES)
 	cd $(OUT_DIR) && $(CARGO_MANIFEST_DIR)/harfbuzz/configure $(CONFIGURE_FLAGS) \
 		CFLAGS="$(CFLAGS)" CXXFLAGS="$(CXXFLAGS)"
 	cd $(OUT_DIR) && make -j$(NUM_JOBS)
 	cd $(OUT_DIR) && make install
+
+# If the timestamp on these files is incorrect (because git does not preserve timestamps)
+# then the configure script will incorrectly try to run automake commands to regenerate them.
+# We `touch` these files to prevent this.
+AUTOMAKE_FILES = \
+	harfbuzz/configure.ac \
+	harfbuzz/aclocal.m4 \
+	harfbuzz/configure \
+	harfbuzz/config.h.in \
+	harfbuzz/Makefile.am \
+	harfbuzz/Makefile.in
 
 .PHONY: all

